### ISSUE TITLE
"You must be of at least an officer in your faction to use this command"

### ISSUE THREAD

**Issue by RoyalRaid** [MarkehMe/FactionsWarps#18]
Hey,
For some strange reason its saying to me "You must be of at least an officer in your faction to use this command" when trying to set a warp for the faction, Im the owner of the fac, therefor the highest rank but cant set warps?

Thanks in advance

**Comment by markhughes** [comment]
Very odd! What version of Factions are you using?

**Comment by RoyalRaid** [comment]
https://www.spigotmc.org/resources/factionsuuid.1035/

**Comment by RoyalRaid** [comment]
Any update?

**Comment by markhughes** [comment]
I've had issues using the FactionsUUID API, I'm in the process of building a solution though. It will just take some time.

**Comment by RoyalRaid** [comment]
Any update?, Would love to release this plugin with my server revamp! Players would love it

**Comment by markhughes** [comment]
I haven't had Factions UUID API working too well, and my attempts in the past to contact the developer have failed. 

I'm building my own maintained spin off of Factions UUID for 1.6 which will be better supported :)

**Comment by RoyalRaid** [comment]
Could i perhaps get the sourcecode to make changes to get it working?

**Comment by markhughes** [comment]
I believe Factions UUID actually has warps already.

Are you just after the additional features that FactionsWarps provides? These are in my revamped version.

**Comment by RoyalRaid** [comment]
yeah FactionsUUID has warps but doesn't have the password feature which is what im after

**Comment by markhughes** [comment]
@RoyalRaid I sent you a message on Spigot if you'd like to test out the new build.

**Comment by jds110** [comment]
Update with this?
